components/brave_ads/core/internal/serving/prediction/model_based/scoring/last_seen/creative_ad_model_based_predictor_last_seen_scoring.cc

```diff
@@ -0,0 +1,25 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_ads/core/internal/serving/prediction/model_based/scoring/last_seen/creative_ad_model_based_predictor_last_seen_scoring.h"
+
+#include "brave/components/brave_ads/core/internal/serving/prediction/model_based/input_variable/last_seen/creative_ad_model_based_predictor_last_seen_input_variable_info.h"
+
+namespace brave_ads {
+
+double ComputeLastSeenScore(
+    const CreativeAdModelBasedPredictorLastSeenInputVariableInfo&
+        last_seen_input_variable) {
+  if (!last_seen_input_variable.value ||
+      last_seen_input_variable.value > base::Days(1)) {
+    return last_seen_input_variable.weight;
```

Comment:
Should this not return 0? I believe this is the condition when the variable value equals 0, so weight * value = 0.

Reply:
No, because if an ad has not been seen after 1 day then the ad is eligible again for serving.

Reply:
Makes sense, I had it backwards; we want to have higher scores for ads not seen within a longer window. Will resolve this comment.